src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -233,6 +233,16 @@ const HoloFuelDnaInterface = {
 
       return noDuplicateIds.filter(tx => tx.status === 'completed').sort((a, b) => a.timestamp > b.timestamp ? -1 : 1)
     },
+    allNewCompleted: async (since) => {
```

Comment:
why not just use the `allCompleted` endpoint, with an optional `since` param?

Reply:
Reverted back to the this method. :+1: